build/org.eclipse.cdt.managedbuilder.core.tests/tests/org/eclipse/cdt/managedbuilder/core/tests/BuildDescriptionModelTests.java

```diff
@@ -1652,13 +1653,6 @@ public void testDesAddlInVarUserObjs() {
 		type.addResource(tDes.createResource("dir2/h.r"));
 		type.addResource(tDes.createResource("dir2/i.r"));
 
-		type = step.createIOType(false, false, null);
-		type.addResource(tDes.createResource("o1.o"));
```

Comment:
Is there no need to check that these files gets handled somewhere else or is it enough to just drop the check?

Reply:
There is nothing further to check in the context of this test since the user objects are no-longer part of the ${INPUTS} of the linker.